apps/postgres-new/components/app-provider.tsx

```diff
@@ -16,11 +18,19 @@ import {
   useRef,
   useState,
 } from 'react'
+import { getTablesQueryKey } from '~/data/tables/tables-query'
 import { DbManager } from '~/lib/db'
 import { useAsyncMemo } from '~/lib/hooks'
-import { isStartupMessage, isTerminateMessage, parseStartupMessage } from '~/lib/pg-wire-util'
-import { parse, serialize } from '~/lib/websocket-protocol'
+import {
+  getMessages,
+  isReadyForQuery,
+  isStartupMessage,
+  isTerminateMessage,
+  parseReadyForQuery,
+  parseStartupMessage,
```

Comment:
I need to continue adding new protocol messages to pg-gateway so we can have those built-in! 😄

Reply:
💯 - I'll help with this too!